.github/workflows/backups.yml

```diff
@@ -34,15 +34,16 @@ jobs:
       matrix: ${{fromJson(needs.get-pipelines.outputs.matrix)}}
       fail-fast: false
 
-    env:
-      TARBALL_FILENAME: ${{ matrix.pipeline }}_$(date +%Y-%m-%d_%H-%M).tar.gz
-
     steps:
-      - name: Clone the repository
+      - name: Clone the nf-core/${{ matrix.pipeline }} repository
         run: git clone --mirror https://github.com/nf-core/${{ matrix.pipeline }}.git
 
       - name: Make a tarball
-        run: tar czf $TARBALL_FILENAME ${{ matrix.pipeline }}.git
+        id: make-tarball
+        run: |
+          TARBALL_FILENAME="${PIPELINE}_$(date +%Y-%m-%d_%H-%M).tar.gz"
```

Comment:
why aren't we using `${{ matrix.pipeline }` in line 44?

Reply:
ah, merge artifact